src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -385,6 +385,7 @@ configurationRegistry.registerConfiguration({
 			policy: {
 				name: 'ChatAgentMode',
 				minimumVersion: '1.99',
```

Comment:
This changes looks good and will work 👍 

If you wanted the policy to set a value other than false, you could use `defaultValue`. `false` is the default though, so this works fine.

https://github.com/microsoft/vscode/blob/01b30db2571c523839c30dfa2d437c6c1a78f749/src/vs/workbench/services/policies/common/accountPolicyService.ts#L64